### PR TITLE
Added error images for Linux OS

### PR DESCRIPTION
**Pull request by kristijanhusak**
Hi,

I added custom icons for Linux OS for both failing and successing. They show up bigger and in colors (green checkmark, or red one-way sign). It can be extended for other operating systems, but i don't know commands for showing different types of errors on those, so this works only for Linux.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 var util = require('util');
 var growl = require('growl');
-var isDarwin = process.platform === 'darwin';
+var os = require('os');
+var isDarwin = process.platform === 'Linux';
```
Comment (alexgb) [inline index.js]: This can't be right

**Comment by alexgb** [comment]
A bit of background... I used emoji in the notification titles so that there would be a visual cue in NotificationCenter notifications on OS X, which don't support the `image` attribute. `isDarwin` is used to conditionally insert the emoji into the message – thus this PR breaks that logic.

This is on the right track, but ideally, we'd have cross platform images for Growl notifications and fallback to emoji on OS X when Growl isn't available.

If you can fix the PR so that it doesn't break NotificationCenter emoji support then I will merge.

**Comment by kristijanhusak** [comment]
Ok i accidentally changed platform check from darwin to Linux. 
I changed it back to darwin and it works ok on OSX with growl, the image attribute does not break the emoji.

**Comment by kristijanhusak** [comment]
Bump. Any chance you merge this ?

**Comment by alexgb** [comment]
Sorry about that, it's now merged.

**Pull request merged**